src/App.tsx

```diff
@@ -17,10 +27,24 @@ function App() {
     addons: new Set(),
   };
 
-  const [step] = useState(0);
+  const [step, setStep] = useState(0);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [isComplete, setIsComplete] = useState(false);
   const [state, dispatch] = useReducer(reducer, initialState);
   const personalInfoFormId = "form-" + useId();
 
+  const goToNextStep = () => {
+    startTransition(() => setStep((step) => step + 1));
```

Comment:
haven't used it yet, but it seems to be nice feature for changing pages that are heavy to load